prometheus/src/main/resources/prometheus.yml

```diff
@@ -30,7 +30,6 @@ scrape_configs:
   - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
-    kubelet_port: 10250
```

Comment:
@jimmidyson you happy for this to go in? I've tested on GKE and it works but perhaps a test on openshift is needed too?